### ISSUE TITLE
Julia package fetcher installed wrong FITSIO version

### ISSUE THREAD

**Issue by rfetick** [JuliaAstro/FITSIO.jl#105]
Hello,

I report here an issue I had with FITSIO installation and reported to [Stackoverflow](https://stackoverflow.com/questions/52274370/cannot-use-fitsio-in-julia/52277557?noredirect=1#comment91514152_52277557). My Julia version seems incompatible with FITSIO since it automatically installed the wrong FITSIO package. I had to manually force the URL to download the correct one. Maybe the problem comes from my Julia version, I just let you know the issue.

Kind regards

**Comment by giordano** [comment]
I don't understand, the UUID of the package that I can read at https://stackoverflow.com/q/52274370/2442087 is correct (`525bcba6-941b-5504-bd06-fd0dc1a4d2eb`).  Can you reproduce the error if you remove the package and try installing again with

```julia
pkg> add FITSIO
```

**Comment by kbarbary** [comment]
I don't think there is a FITSIO release that supports Julia 1.0 yet, is
there? If not, you need the development version, which you can get with

pkg> dev FITSIO

**Comment by rfetick** [comment]
I agree it is quite strange...

When looking into the FITSIO.jl files the major difference is following. 
Wrong file

*|type ImageHDU <: HDU fitsfile::FITSFile ext::Int end|*

and replaced by (correct file)

*mutable struct ImageHDU <: HDU****     fitsfile::FITSFile****     ext::Int****end*

Moreover the path to the package is different

*packages/FITSIO/WueGk/* (wrong one)

*packages/FITSIO/owJoM/* (good one)

I just tried to remove the package and add it again with *add FITSIO*, 
same error. Again I solved the problem by setting your github URL. For 
information, I am working on windows...

Romain


Le 12/09/2018 à 21:29, Mosè Giordano a écrit :

**Comment by giordano** [comment]
Kyle is correct: the latest stable version of this package isn't compatible with Julia 1.0. This has been already fixed in the development version, but we haven't tagged a new version since then. The fact that the paths are different is normal, there is nothing strange with it, the package manager allows different versions to coexist.

Please, remove FITSIO.JL with
```julia
pkg> rm FITSIO
pkg> update
pkg> add FITSIO
pkg> dev FITSIO
```
After the rm, make sure you don't have any FITSIO.jl leftover, then you can reinstall and switch to the development version within the package manager, without installing another package

**Comment by giordano** [comment]
The problem for me to use the development version of the package is that I didn't realize that there wasn't a stable version supporting julia 1.0 :sweat_smile: thanks to Kyle for pointing this out.

I'm going to tag a new version later today.

**Comment by giordano** [comment]
This can be closed, v0.12.0 has been available for a while now
